### ISSUE TITLE
Scan Net HTTPNotFound

### ISSUE THREAD

**Issue by MaximusMcCann** [fastlane/fastlane#7941]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description

Sometimes my tests fail on Circle CI, wondering if this random error has anything to do with it.  See ❌---***What's this?***---❌  below.  Cheers!

##### Complete output when running fastlane, including the stack trace and command used
> You can use: `--capture_output` as the last commandline argument to get that collected for you

```
➜  ios: bundle exec fastlane test
+------------------------+---------+--------------------+
|                     Used plugins                      |
+------------------------+---------+--------------------+
| Plugin                 | Version | Action             |
+------------------------+---------+--------------------+
| fastlane-plugin-sentry | 0.1.2   | sentry_upload_dsym |
+------------------------+---------+--------------------+

[13:38:34]: -----------------------------
[13:38:34]: --- Step: update_fastlane ---
[13:38:34]: -----------------------------
[13:38:34]: Looking for updates for fastlane...
[13:38:36]: Nothing to update ✅
[13:38:36]: ------------------------------
[13:38:36]: --- Step: default_platform ---
[13:38:36]: ------------------------------
[13:38:36]: Driving the lane 'test' 🚀
[13:38:36]: ------------------
[13:38:36]: --- Step: scan ---
[13:38:36]: ------------------
[13:38:36]: Successfully loaded '/Users/a/CODE/ios/fastlane/Scanfile' 📄

+--------------+----------------------------------------------------------+
|               Detected Values from './fastlane/Scanfile'                |
+--------------+----------------------------------------------------------+
| scheme       | client                                                   |
| devices      | ["iPhone 5 (9.0)", "iPhone 5 (10.0)", "iPhone 7 (10.1)"] |
| open_report  | false                                                    |
| clean        | true                                                     |
| output_types | html, junit                                              |
+--------------+----------------------------------------------------------+
...
... //Compile, etc
...
[13:53:27]: ▸ BuildConfiurationTests
[13:53:27]: ▸ ✓ testBuildConfigurationIsCorrect (0.001 seconds)
[13:53:27]: ▸ 	 Executed 3 tests, with 0 failures (0 unexpected) in 0.007 (0.017) seconds
[13:53:27]: ▸
[13:53:31]: ▸ Test Succeeded
[13:53:36]: #<Net::HTTPNotFound:0x007f8a9bfa5fc8>  //❌---***What's this?***---❌
+--------------------+---+
|      Test Results      |
+--------------------+---+
| Number of tests    | 3 |
| Number of failures | 0 |
+--------------------+---+

[13:53:41]: Successfully generated report at '/Users/a/CODE/ios/fastlane/test_output/report.html'
[13:53:45]: Successfully generated report at '/Users/a/CODE/ios/fastlane/test_output/report.junit'

+------+------------------+-------------+
|           fastlane summary            |
+------+------------------+-------------+
| Step | Action           | Time (in s) |
+------+------------------+-------------+
| 1    | update_fastlane  | 2           |
| 2    | default_platform | 0           |
| 3    | scan             | 909         |
+------+------------------+-------------+

[13:53:45]: fastlane.tools just saved you 15 minutes! 🎉
```

### Environment

Please run `fastlane env` and copy the output below. This will help us help you :+1:
If you used `--capture_output` option please remove this block - as it is already included there.

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                 | Value                                       |
| ------------------- | ------------------------------------------- |
| OS                  | 10.12.2                                     |
| Ruby                | 2.3.1                                       |
| Bundler?            | true                                        |
| Git                 | git version 2.9.3 (Apple Git-75)            |
| Installation Source | ~/.rbenv/versions/2.3.1/bin/fastlane        |
| Host                | Mac OS X 10.12.2 (16C67)                    |
| Ruby Lib Dir        | ~/.rbenv/versions/2.3.1/lib                 |
| OpenSSL Version     | OpenSSL 1.0.2j  26 Sep 2016                 |
| Is contained        | false                                       |
| Xcode Path          | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version       | 8.1                                         |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
desc "Runs all the tests"
lane :test do
  scan
end
```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.9.0   | ✅ Up-To-Date  |

### Loaded fastlane plugins:

| Plugin                 | Version | Update-Status |
| ---------------------- | ------- | ------------- |
| fastlane-plugin-sentry | 0.1.2   | ✅ Up-To-Date  |

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.0.2        |
| bundler                   | 1.13.6       |
| io-console                | 0.4.6        |
| CFPropertyList            | 2.3.4        |
| i18n                      | 0.7.0        |
| json                      | 1.8.6        |
| minitest                  | 5.10.1       |
| thread_safe               | 0.3.5        |
| tzinfo                    | 1.2.2        |
| activesupport             | 4.2.7.1      |
| public_suffix             | 2.0.5        |
| addressable               | 2.5.0        |
| babosa                    | 1.0.2        |
| claide                    | 1.0.1        |
| fuzzy_match               | 2.0.4        |
| nap                       | 1.1.0        |
| cocoapods-core            | 1.1.0        |
| cocoapods-deintegrate     | 1.0.1        |
| cocoapods-downloader      | 1.1.3        |
| cocoapods-plugins         | 1.0.0        |
| cocoapods-search          | 1.0.0        |
| cocoapods-stats           | 1.0.0        |
| netrc                     | 0.7.8        |
| cocoapods-trunk           | 1.1.2        |
| cocoapods-try             | 1.1.0        |
| colored                   | 1.2          |
| escape                    | 0.0.4        |
| fourflusher               | 2.0.1        |
| gh_inspector              | 1.0.3        |
| molinillo                 | 0.5.5        |
| nanaimo                   | 0.2.3        |
| xcodeproj                 | 1.4.2        |
| cocoapods                 | 1.1.0        |
| highline                  | 1.7.8        |
| commander                 | 4.4.3        |
| unf_ext                   | 0.0.7.2      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20161129 |
| dotenv                    | 2.1.2        |
| excon                     | 0.54.0       |
| multipart-post            | 2.0.0        |
| faraday                   | 0.11.0       |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| faraday_middleware        | 0.11.0       |
| fastimage                 | 2.0.1        |
| jwt                       | 1.5.6        |
| little-plugger            | 1.1.4        |
| multi_json                | 1.12.1       |
| logging                   | 2.1.0        |
| memoist                   | 0.15.0       |
| os                        | 0.9.6        |
| signet                    | 0.7.3        |
| googleauth                | 0.5.1        |
| httpclient                | 2.8.3        |
| hurley                    | 0.2          |
| mime-types                | 1.25.1       |
| uber                      | 0.0.15       |
| representable             | 2.3.0        |
| retriable                 | 2.1.0        |
| google-api-client         | 0.9.20       |
| mini_magick               | 4.5.1        |
| multi_xml                 | 0.6.0        |
| plist                     | 3.2.0        |
| rubyzip                   | 1.2.0        |
| security                  | 0.1.3        |
| slack-notifier            | 1.5.1        |
| terminal-notifier         | 1.7.1        |
| unicode-display_width     | 1.1.3        |
| terminal-table            | 1.7.3        |
| word_wrap                 | 1.0.0        |
| rouge                     | 1.11.1       |
| xcpretty                  | 0.2.4        |
| xcpretty-travis-formatter | 0.0.4        |
| rest-client               | 1.6.9        |
| fastlane-plugin-sentry    | 0.1.2        |
</details>

*generated on:* **2017-01-20**
</details>

**Comment by mfurtak** [comment]
See #8004 for an explanation of that error message. If that satisfies the point of this ticket, please close it. Otherwise, could you rephrase this ticket? Thanks!

**Comment by asfalcone** [comment]
Going to close this issue since there hasn't been a reply in some time. Please see #8004 for an explanation of that error message. If you're still having a problem, please open a new issue with more details. Thanks!
